resources/views/email-verification/emails/text.blade.php

```diff
@@ -0,0 +1,11 @@
+Hello,
+
+We had request to register account with this email.
```

Comment:
Think we might have to figure out another default text e-mail.

In my mind, you're not _requesting_ permission to register an account. You're just asked to verify that you've registed an account with the given e-mail.